python/hail/methods/impex.py

```diff
@@ -764,13 +764,179 @@ def import_gen(path, sample_file=None, tolerance=0.2, min_partitions=None, chrom
            reference_genome=nullable(GenomeReference))
 def import_table(paths, key=[], min_partitions=None, impute=False, no_header=False,
                  comment=None, delimiter="\t", missing="NA", types={}, quote=None, reference_genome=None):
+    """Import delimited text file (text table) as key table.
+
+    The resulting key table will have no key columns, use :meth:`.KeyTable.key_by`
+    to specify keys.
+
+    Examples
+    --------
+
+    Given this file
+
+    .. code-block:: text
+
+        $ cat data/samples1.tsv
+        Sample     Height  Status  Age
+        PT-1234    154.1   ADHD    24
+        PT-1236    160.9   Control 19
+        PT-1238    NA      ADHD    89
+        PT-1239    170.3   Control 55
+
+    The interesting thing about this table is that column ``Height`` is a
+    floating-point number, and column ``Age`` is an integer. Hail can impute
+    these types or we can state the explicitly:
+
+    Importing a table with explicit type annotations:
+
+    >>> table = hc1.import_table('data/samples1.tsv', types={'Height': TFloat64(), 'Age': TInt32()})
+
+    Note that string columns like ``Sample`` and ``Status`` do not need to be
+    typed, because ``String`` is the default type.
+
+    Imputing a table with type imputation (the file must be read twice to
+    impute):
+
+    >>> table = hc1.import_table('data/samples1.tsv', impute=True)
+
+    **Detailed examples**
+
+    Let's import annotations from a CSV file with missing data and special characters:
+
+    .. code-block:: text
+
+        $ cat data/samples2.tsv
+        Batch,PT-ID
+        1kg,PT-0001
+        1kg,PT-0002
+        study1,PT-0003
+        study3,PT-0003
+        .,PT-0004
+        1kg,PT-0005
+        .,PT-0006
+        1kg,PT-0007
+
+    In this case, we should:
+
+    - Pass the non-default delimiter ``,``
+
+    - Pass the non-default missing value ``.``
+
+    >>> table = hc1.import_table('data/samples2.tsv', delimiter=',', missing='.')
+
+    Let's import annotations from a file with no header and sample IDs that need
+    to be transformed.  Suppose the vds sample IDs are of the form
+    ``NA#####``. This file has no header line, and the sample ID is hidden in a
+    field with other information.
+
+    .. code-block: text
+
+        $ cat data/samples3.tsv
+        1kg_NA12345   female
+        1kg_NA12346   male
+        1kg_NA12348   female
+        pgc_NA23415   male
+        pgc_NA23418   male
+
+    To import:
+
+    >>> annotations = (hc1.import_table('data/samples3.tsv', no_header=True)
+    ...                   .annotate('sample = f0.split("_")[1]')
+    ...                   .key_by('sample'))
+
+    **Notes**
+
+    The `impute` parameter tells Hail to scan the file an extra time to gather
```

Comment:
I think we've standardized on double back-ticks for parameters, here and below

Reply:
We standardized on single-backticks. Can you confirm @tpoterba ?

Reply:
Yeah, I think you're right

Reply:
Yep!
The **Notes** should be a 
```
Notes
-----
```
too